scripts/gcs/a6000/startup_picture_serve_and_take.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -
+
+A6000_PATH=/home/odroid/Desktop/SUAS-Competition/scripts/gcs/a6000
+
+cd /home/odroid/Desktop/SUAS-Competition/scripts/gcs/a6000
+
+python $A6000_PATH/picture_server.py &
+sleep 10
+bash $A6000_PATH/picture_taker.sh
```

Comment:
they python and bash paths should relative since we have used cd to move into the directory